src/ContinuousDynamicalSystem/src/FixedBaseDynamics.cpp

```diff
@@ -32,28 +32,65 @@ bool FixedBaseDynamics::initialize(std::weak_ptr<IParametersHandler> handler)
 
     if (!ptr->getParameter("gravity", m_gravity))
     {
-        log()->info("{} The gravity vector  not found. The default one will be "
+        log()->info("{} The gravity vector is not found. The default one will be "
                     "used {}.",
                     logPrefix,
                     m_gravity.transpose());
     }
 
+    if (ptr->getParameter("base_frame", m_robotBase))
+    {
+        if (m_kinDyn.isValid())
+        {
+            if (!m_kinDyn.setFloatingBase(m_robotBase))
```

Comment:
`setFloatingBase` only support links, not frames at the moment (see https://github.com/robotology/idyntree/issues/422). I think this should be reflected in the parameter docs, and perhaps also in its name.

Reply:
Done in  https://github.com/dic-iit/bipedal-locomotion-framework/pull/275/commits/42fc31128d938e3b3d734fd7d99d6820c6597116 and https://github.com/dic-iit/bipedal-locomotion-framework/pull/275/commits/7beba8c3821fae01498895bc8e695db7a5a502fd